tests/MenderAPI/deviceconnect.py

```diff
@@ -48,3 +48,22 @@ def get_websocket(self):
         )
 
         return ws
+
+    def get_playback_url(self, session_id, sleep_ms=None):
+        url = "wss://%s/api/management/%s/deviceconnect/sessions/%s/playback" % (
```

Comment:
arent we putting the urls in a global variables somewhere usually? (as far as I remember, but I maybe wrong)

Reply:
This was a copy/paste/modify of the other class member function.